app/views/new_event_mailer/approve_volunteer.html.slim

```diff
@@ -0,0 +1,12 @@
+h4 
+  | Hello 
+  = @volunteer.name
+
+p Congratulations! Your volunteer application has been approved. We will be following up with more details. Stay tuned!
+
+h2 = @event.event_name
+h2 = @event.date
+
+| Find out more about the event at: 
+= link_to 'Event page', event_url(@event)
```

Comment:
nice! almost everyone gets this wrong, and uses `_path` which is relative path. using `_url` will use the whole domain path.